docker-compose.yml

```diff
@@ -7,7 +7,7 @@ x-logging: &log-journald
 
 services:
   api:
-    image: ${ORG}/montagu-api:master
+    image: ${ORG}/montagu-api:abf39a2
```

Comment:
This is currently pointed at https://github.com/vimc/montagu-api/pull/479, I'll update once that PR is merged.